browser-test-harness.js

```diff
@@ -9,29 +9,52 @@ const storybook = childProcess.spawn(process.execPath, [
   '9002',
 ]);
 
+const standalone = childProcess.spawn(process.execPath, [
+  path.join('test', 'standalone', 'start.sh'),
+  '9003',
+]);
+
 process.on('exit', () => {
   storybook.kill();
+  standalone.kill();
 });
 
 waitPort({
```

Comment:
can you explain this change?

Reply:
standalone is a mini webpack server and client instance, the nonce has to be generated at the server